Document/0x04a-Mobile-App-Taxonomy.md

```diff
@@ -1,32 +1,40 @@
 ## Mobile App Taxonomy
 
-The term "mobile app" refers to self-contained computer programs that are designed to execute on mobile devices .Today, mobile Internet usage has surpassed desktop usage for the first time in history, and mobile apps are the [most widespread kind of applications](http://www.idc.com/promo/smartphone-market-share/os "Worldwide Smartphone OS Market Share"). In this guide, we focus on mobile apps designed to run on the Android and iOS operating systems, which cumulatively take [more than 99% of the mobile OS market share](http://bgr.com/2016/11/02/internet-usage-desktop-vs-mobile "Mobile Internet usage surpasses Desktop usage for the first time in history"). These apps don't necessarily always run only on mobile devices - they are increasingly used on other device types, such as smart watches, TVs, cars, and embedded systems. In this guide, we'll be using the term "app" to refer to any kinds of apps running on popular mobile OSes.
+The term "mobile app" refers to a self-contained computer program, designed to execute on a mobile device. Today, the Android and iOS operating systems cumulatively compose [more than 99% of the mobile OS market share] (http://www.idc.com/promo/smartphone-market-share/os). Additionally, mobile Internet usage has surpassed desktop usage for the first time in history, making mobile browsing and apps the [most widespread kind of Internet-capable applications] (http://www.idc.com/promo/smartphone-market-share/os), considering the market saturation of smart devices.
```

Comment:
make sure that the references don't have a space in between e.g.

market share] (http:  -> market share](http: